assets/post.css

```diff
@@ -210,6 +214,12 @@ header {
   background-size: cover;
   background-repeat: no-repeat;
 }
+#bg3 {
+  background: url("/assets/partIII-cover-image.jpg");
```

Comment:
same RE: above, it seems like we follow "partN" the most (e.g. part3) so could we follow that as all the figure names are already in that pattern